.github/workflows/unitTest.yml

```diff
@@ -28,18 +23,8 @@ jobs:
       - name: Install
         run: bun install
 
-      - name: Format
-        run: deno fmt --check
-
-      - name: Lint
-        run: deno lint
-
       - name: Bundle
         run: bun bun
```

Comment:
Maybe bundling should be a separate workflow, too.

Reply:
Done